smoke-tests/designer/wdio.conf.js

```diff
@@ -1,6 +1,6 @@
 const { hooks } = require("./support/hooks");
 const drivers = {
-  chrome: { version: "102.0.5005.61" }, // https://chromedriver.chromium.org/
+  chrome: { version: "104.0.5112.79" }, // https://chromedriver.chromium.org/
```

Comment:
This presumably is going to keep breaking as Chrome updates, so we should try to find a more sustainable way to keep this updated or in-sync with CI.

Reply:
Yeah i think there needs to be a task to auto update this rather than having to manually change it